raitools/tests/test_raianalyzer.py

```diff
@@ -50,18 +54,38 @@ def test_raianalyzer_binary(self):
 
 
 def run_raianalyzer(model, x_train, x_test, target_column, classes):
-    error_analyzer = RAIAnalyzer(model, x_train, x_test, target_column,
-                                 task_type=ModelTask.CLASSIFICATION)
-    error_analyzer.explainer.add()
+    raianalyzer = RAIAnalyzer(model, x_train, x_test, target_column,
+                              task_type=ModelTask.CLASSIFICATION)
+    raianalyzer.explainer.add()
     # Validate calling add multiple times prints a warning
     with pytest.warns(UserWarning):
-        error_analyzer.explainer.add()
-    error_analyzer.explainer.compute()
-    explanations = error_analyzer.explainer.get()
+        raianalyzer.explainer.add()
+    raianalyzer.explainer.compute()
+    validate_explainer(raianalyzer, x_train, x_test, classes)
+    with TemporaryDirectory() as tempdir:
+        path = Path(tempdir) / 'explanation'
+        # save the raianalyzer
+        raianalyzer.save(path)
+        # load the raianalyzer
+        raianalyzer = RAIAnalyzer.load(path)
```

Comment:
Would be nice to check that you got back what you saved.

Reply:
that's what validate_explainer() is doing in the test, each of the managers can be validated on raianalyzer after it's loaded back

Reply:
added more validation for raianalyzer train/test dataset, task type and target column before and after serialization

Reply:
But what about the rest of the `RAIAnalyzer` - the properties on the root object itself, for example. Plus, does the validation mentioned check that it got back exactly what it saved?

Reply:
yes, exactly, I just added that validation in latest update